posthog/tasks/periodic_digest.py

```diff
@@ -68,8 +81,22 @@ def get_teams_with_new_event_definitions(end: datetime, begin: datetime) -> Quer
     return EventDefinition.objects.filter(created_at__gt=begin, created_at__lte=end).values("team_id", "name", "id")
 
 
-def get_teams_with_new_playlists(end: datetime, begin: datetime) -> QuerySet:
-    return (
+def get_teams_with_new_playlists(end: datetime, begin: datetime) -> list[CountedPlaylist]:
```

Comment:
this is relatively messy but we can tidy it up if we see that folk interact with these links in the email